bot/exts/utilities/bookmark.py

```diff
@@ -87,8 +86,8 @@ async def send_reaction_embed(
         await message.add_reaction(BOOKMARK_EMOJI)
         return message
 
-    @whitelist_override(channels=WHITELISTED_CHANNELS, categories=WHITELISTED_CATEGORIES)
     @commands.command(name="bookmark", aliases=("bm", "pin"))
```

Comment:
missing a trailing comma in aliases tuple

Reply:
This shouldn't have a trailing comma. You only use a trailing comma here when there's one alias (to make it a tuple).

Reply:
Trailing commas are only for multiline.